security/pkg/credentialfetcher/plugin/gce.go

```diff
@@ -0,0 +1,72 @@
+// Copyright Istio Authors
```

Comment:
We have GCP/GKE/google specific code spread in too many places. 

Please find the others - and use one, preferably under pkg/... - it's not sustainable for each vendor to add packages all over the place, plus we have the same problem we had with K8S, inconsistencies and duplication.

Reply:
But this duplicates code already present in gRPC library we depend on - which also has multiple implementations for getting tokens, perfectly fit for our needs. 

So why would we use a different interface - and have to reimplement and maintain this code ?

Reply:
Resolving it based on the consensus mentioned in other comments.